tests/benchmarks/longmemeval/test_longmemeval.py

```
@@ -0,0 +1,763 @@
+                # Flatten: AutoMem nests memory data under "memory" key
+                flattened = []
+                for r in results:
+                    mem = r.get("memory", {})
+                    mem["score"] = r.get("score", 0)
+                    mem["match_type"] = r.get("match_type", "")
+                    flattened.append(mem)
+                return flattened
+            else:
+                print(f"  Recall failed for {question_id}: {response.status_code}")
+                return []
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)